apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/network/AppSync.kt

```diff
@@ -0,0 +1,46 @@
+package com.apollographql.apollo3.network
+
+import com.apollographql.apollo3.api.http.DefaultHttpRequestComposer.Companion.appendQueryParameters
+import com.apollographql.apollo3.api.internal.json.BufferedSinkJsonWriter
+import com.apollographql.apollo3.api.internal.json.Utils
+import okio.Buffer
+
+object AppSync {
+
+  /**
+   * Helper method that builds the final URL. It will append the authorization and payload arguments as query parameters.
+   * This method can be use for both the HTTP URL as well as the WebSocket URL
```

Comment:
```suggestion
   * This method can be used for both the HTTP URL as well as the WebSocket URL
```

Reply:
Also, I'm still confused by `https://` vs `wss://` is there a reason to use one over the other? If not we could add a note about it? 

```
Note: https:// and wss:// urls (and http:// and ws:// for unsecure connections) are treated in the same way be the underlying code. Using one host or the other does not matter.
```